.github/workflows/prbuild.yaml

```diff
@@ -6,8 +6,36 @@ on: [push, pull_request]
 env:
   GOPROXY: https://proxy.golang.org/
 jobs:
-  integration:
+  integration-latest:
     runs-on: ubuntu-latest
+    env:
+      NODEIMAGE: docker.io/kindest/node:v1.20.2
+    steps:
```

Comment:
Oh just occurred to me could we use YAML anchors/references to DRY this out?

Reply:
did some googling, AFAICT GH Actions does not support YAML anchors: https://github.community/t/support-for-yaml-anchors/16128

Reply:
ah thats no good :(

Reply:
maybe we could use this? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
could make it so we only have 1 "job" still but different steps within that

Reply:
Switched to a matrix and it runs well. The only annoying thing is that now in branch protection rules, the required job names include the specific node image value, so we'll have to change those every time we change a Kube version we run against. Looking to see if there's a way around that..

Reply:
oh that is very annoying 👎🏽